test/integration/instance_with_advanced_options/inspec.yml

```diff
@@ -0,0 +1,27 @@
+name: simple_instance
+attributes:
+  - name: project_id
+    required: true
+    type: string
+
+  - name: zone
+    required: true
+    type: string
+
+  - name: instance_name
+    required: true
+    type: string
+
+  - name: network
+    required: false
+    default: "default"
```

Comment:
Defaults should not be used to ensure outputs are correctly wired to attributes.